stl/inc/vector

```diff
@@ -515,6 +515,12 @@ private:
 
     _CONSTEXPR20 void _Modify_annotation(const difference_type _Count) const noexcept {
         // Extends/shrinks the annotated range by _Count
+        if (_Count == 0) { // nothing to do
+            // This also avoids calling _Apply_annotation() with null pointers

```

Comment:
Can we also change the `_Unfancy_maybe_null` calls in `_Apply_annotation` to `_Unfancy`s?

Reply:
Seems possible but I would like to create a followup issue because that's a "nice to have" that could delay this BE RI fix.

Reply:
Since this is a suggested simplification and not a correctness issue, I've approved and made the PR Ready To Merge despite the open question. We can always followup with this change if the PR is merged before @cubezault has time to consider it.

Reply:
> Seems possible but I would like to create a followup issue because that's a "nice to have" that could delay this BE RI fix.

Jinx!

Reply:
Filed a followup issue. Please meow if you believe we should re-spin the PR instead.